ci/tests/puppeteer/run.sh

```diff
@@ -97,6 +102,14 @@ while (( "$#" )); do
     BUILDFLAGS="${BUILDFLAGS} --offline"
     shift 1;
     ;;
+  --max-workers|--mw)
```

Comment:
Likewise, this should either be removed, or the value should be fixed and not customizable. I don't ever see the need by anyone to adjust this.